slitlessutils/core/preprocess/crrej/drizzle.py

```diff
@@ -0,0 +1,70 @@
+from drizzlepac import astrodrizzle
+from pathlib import Path
+
+# Args taken from INS HSTaXe FullFrame Cookbooks
+COMMON_ARGS = {
+    'ir': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': False,
+        'final_fillval': None
+    },
+    'uvis': {
+        'mdriztab': True,
+        'preserve': False,
+        'skysub': True,
+        'skymethod': 'match',
+        'final_fillval': None
+    },
+    'acs': {
+        'build': True,
+        'mdriztab': True,
+        'in_memory': False,
+        'skysub': False,
+        'driz_separate': False,
+        'median': False,
+        'blot': False,
+        'driz_cr': False,
+        'driz_sep_wcs': False,
+        'driz_combine': True,
+        'final_wcs': False
+    }
+}
+
+
+def drizzle(files, instrument=None, outdir=Path().absolute(), **kwargs):
+    '''
+    Runs AstroDrizzle as part of Cosmic Ray handling. Passes any additional arguments
+    not listed here directly to AstroDrizzle. Any user-specified arguments will override
+    
+
+    Parameters
+    ----------
+    files : str to a file catalog, or a python list of files
+        The list of file to be processed by AstroDrizzle
+    
+    instrument : str (one of 'ir', 'uvis', 'acs')
+        One of three instruments on HST of which to apply a set of default arguments
+        captured from the official HSTaXe FullFrame Cookbooks
+
+    outdir : str or `pathlib.Path`
+        A directory to write the final rectified mosaics to.
+        By default, the current working directory
+    '''
+    # Start with known defaults
+    drizzle_kwargs = {
+        'output': 'su_drizzle',
+        'overwrite': False,
+        'preserve': False,
+        'clean': True
+    }
+    # Apply instrument-specific defaults
+    drizzle_kwargs.update(COMMON_ARGS.get(instrument, {}))
```

Comment:
Thanks @haticekaratay! So I implicitly built that in here. `.get` attempts to grab whatever the user requests here, and if it can't find it, then it passes the fallback, which I specified as an empty dict. That way, if the user supplies something that isn't recognized, it won't crash. Unless you want to explicitly raise a warning?

Reply:
If `instrument` is not one of the allowed values, a `ValueError` should be raised.

Reply:
Yes, @duytnguyendtn, If the intended purpose of this function is to select from a specific set of instruments, any other action should result in an error message or a clear warning for the user.

Reply:
Done in https://github.com/spacetelescope/slitlessutils/pull/27/commits/17bcea48d74d256a4cb3c2295b272fe6985b42b7